internal/clearingway/handlers.go

```diff
@@ -307,20 +307,19 @@ func (c *Clearingway) InteractionCreate(s *discordgo.Session, i *discordgo.Inter
 		case MenuRemove:
 			switch CommandType(command[1]) {
 			case CommandMenu:
-				// send_menu()
+				c.MenuStaticRespond(s, i, command[0])
```

Comment:
just like the other comment in the PR let's accurately describe what command[0] is in a variable name